src/Java/Test.java

```diff
@@ -0,0 +1,7 @@
+package Java;
+
+public class Test {
+    public static void main(String[] args) {
+        System.out.println("Just test");
+    }
```

Comment:
Тест работает. Теперь просто поменяй стрингу в тесте☺
```suggestion

public class Test {
    public static void main(String[] args) {
        System.out.println("Final step");
    }
```

Reply:
Так а почему ты поменяла стрингу не так как я указала?)